tozny/e3db-clients-go
# [WR-release] identityClient_test stability improvements

Improve identityClient_test stability

When running against the development environment tests can randomly fail

- re-order identity test methods, move all helpers to the top of the file
  - this makes it easier to comment out large amount of tests if needed
- increase test suite timeout (20 minutes instead of default 10 minutes)
  - avoid `panic: test timed out after 10m0s` errors when running against development
- refactor uniqueString helper to use random int instead of timestamp
  - ensure we use this helper for all realm names
- add retry for realm creation
  - currently retries once after failure
  - so far in all my test runs a single retry appears to be enough

![image](https://user-images.githubusercontent.com/52154/131534844-50d312cc-533e-40e8-9a6b-870a6e1a36d9.png)
